### PR TITLE
[YUNIKORN-1383] [website] REST API doc changes in Yunikorn Website

### PR DESCRIPTION
**Pull request by zhuqi-lucas**
### What is this PR for?
A few sentences describing the overall goals of the pull request's commits.
First time? Check out the contributing guide - http://yunikorn.apache.org/community/how_to_contribute   

### What type of PR is it?
* [ ] - Bug Fix
* [ ] - Improvement
* [ ] - Feature
* [x] - Documentation
* [ ] - Hot Fix
* [ ] - Refactoring

### Todos
* [ ] - Task

### What is the Jira issue?
* Open an issue on Jira https://issues.apache.org/jira/browse/YUNIKORN/
* Put link here, and add [YUNIKORN-*Jira number*] in PR title, eg. `[YUNIKORN-2] Gang scheduling interface parameters`

### Questions:
* [ ] - The licenses files need update.
* [ ] - There is breaking changes for older versions.
* [ ] - It needs documentation.

### CODE REVIEW

**Review by 0yukali0 — APPROVED** [review]
LGTM
The route links are right.
I built the website successfully.

File: docs/api/scheduler.md [hunk 1]
```diff
@@ -681,6 +681,353 @@ Fetch an Application given a Partition, Queue and Application ID and displays ge
...
+```json
+[
+  {
+    "userName": "user1",
+    "groups": {
+      "app2": "tester"
+    },
+    "queues":
+    {
+      "queuename": "root",
```
[the remark this answers is not in the document]
Comment (wilfred-s) [reply docs/api/scheduler.md:710]: This is not correct in the example output and should be fixed, it really is `queuePath`

**Comment by zhuqi-lucas** [comment]
Thank you @wilfred-s @0yukali0 for review, addressed the comments in latest PR.

**Review by wilfred-s — APPROVED** [review]
LGTM, @0yukali0 any further comments?

**Review by 0yukali0 — APPROVED** [review]
LGTM 😄

**Pull request merged**